rest_framework_swagger/views.py

```diff
@@ -23,7 +24,12 @@ class SwaggerSchemaView(APIView):
         ]
 
         def get(self, request):
-            generator = SchemaGenerator(
+            if schema_generator_cls is None:
+                generator_cls = SchemaGenerator
+            else:
+                generator_cls = schema_generator_cls
```

Comment:
Why not to make `SchemaGenerator` default parameter value instead of this condition?